src/ACTSSeededCKFTrackingProc.cxx

```diff
@@ -131,84 +131,92 @@ ACTSSeededCKFTrackingProc::ACTSSeededCKFTrackingProc() : ACTSProcBase("ACTSSeede
                              _seedFinding_minPt,
                              _seedFinding_minPt);
 
+  registerProcessorParameter("SeedFinding_impactMax",
```

Comment:
Could you please upper case the first letter `"SeedFinding_ImpactMax"` for consistency with the other settings?